### PR TITLE
Mock Arkivo in Items Controller Spec for speed.

### PR DESCRIPTION
**Pull request by grosscol**
Decouple specs from AF,Solr, and Arkivo calls.

### CODE REVIEW

**Comment by grosscol** [comment]
This whole PR is still a mess.  It can stand some more DRYing and contemplation of testing the view rendered.

**Comment by grosscol** [comment]
Apparently I stopped covering all of our codebase with this.  So testing the associated views might  be more important that I anticipated.

File: app/controllers/api/items_controller.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
+require 'pry'
 module API
   # Adds an endpoint that consumes and RESTfully emits JSON from Arkivo
   # representing new and updated Zotero-managed publications. An item in the
-  # Zotero parlance is mapped to a GenericFile in Sufia.
+  # Zotero parlance is mapped to a Work in Sufia.
```
Comment (mjgiarlo) [inline app/controllers/api/items_controller.rb]: `GenericWork` I believe, for better or worse.

**Comment by mjgiarlo** [comment]
You'll want to rebase this on `master` once #1490 is merged.

**Comment by grosscol** [comment]
Rebased and ready for review.

**Pull request merged**